docs/source/general/settings.rst

```diff
@@ -151,9 +151,26 @@ with other freeradius modules.
 
 Set this to ``True`` if you are performing authorization exclusively through the REST API.
 
-Batch mail settings
-~~~~~~~~~~~~~~~~~~~
+Sending emails to users
+-----------------------
 
-    * `DJANGO_FREERADIUS_BATCH_MAIL_SUBJECT <importing_users.html#django-freeradius-batch-mail-subject>`_
-    * `DJANGO_FREERADIUS_BATCH_MAIL_MESSAGE <importing_users.html#django-freeradius-batch-mail-message>`_
-    * `DJANGO_FREERADIUS_BATCH_MAIL_SENDER <importing_users.html#django-freeradius-batch-mail-sender>`_
+Emails can be sent to users whose usernames or passwords have been autogenerated. The content of these emails are customisable. Many settings have been defined for doing that:
+
+``DJANGO_FREERADIUS_BATCH_MAIL_SUBJECT``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is the subject of the mail to be sent to the users. Eg: ``Login Credentials``
```

Comment:
specify the default value as it's done for the other settings (find it here: https://github.com/openwisp/django-freeradius/blob/master/django_freeradius/settings.py#L23-L24)